README.md

```diff
@@ -43,6 +43,10 @@ For available configuration settings, please refer to [Grafana Image Rendering d
 
 Instead of installing and running the image renderer as a plugin, you can run it as a remote image rendering service using Docker. Read more about [remote rendering using Docker](https://github.com/grafana/grafana-image-renderer/blob/master/docs/remote_rendering_using_docker.md).
 
+### Remote Rendering in Kubernetes
```

Comment:
We use sentence case in [titles](https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-style-guide.md#headings).
```suggestion
### Remote rendering in Kubernetes
```